x-pack/plugin/runtime-fields/qa/src/main/java/org/elasticsearch/xpack/runtimefields/test/CoreTestTranslater.java

```diff
@@ -118,15 +90,11 @@ private static String painlessToLoadFromSource(String name, String type) {
 
     // TODO there isn't yet a way to create fields in the runtime section from a dynamic template
     protected static Map<String, Object> dynamicTemplateToAddRuntimeFields(String type) {
-        return Map.ofEntries(
-            Map.entry("type", "runtime"),
-            Map.entry("runtime_type", type),
-            Map.entry("script", painlessToLoadFromSource("{name}", type))
-        );
+        return Map.ofEntries(Map.entry("type", "runtime"), Map.entry("runtime_type", type));
```

Comment:
this seems wrong but it's irrelevant because the tests that use it are disabled. I plan on replacing the dynamic template with dynamic:runtime